set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
```

Comment:
Can pipelines be run from data loaded in the Genomics API?  That was a concern of mine that the Pipeline API bypasses the Genomics API, which could benefit from more integration as all that processed data is not organized and properly searchable, which would become a bigger missed opportunity down the line.  I posted this at the following link:

https://github.com/googlegenomics/pipelines-api-examples/issues/13#issuecomment-195080153